exercises/strings/capitalize/capitalize.js

```diff
@@ -11,14 +11,14 @@
  */
 
 function capitalize(num) {
-  // This is your job. :)
+  return string = num,
```

Comment:
Hah, so, I did a bad thing naming the argument `num`. The name is just a placeholder. But, you could change your code to work as follows:

```js
function capitalize(num) {
  let string = num;
  return string.charAt(0).toUpperCase() + string.slice(1,7);
}
```

We create a new variable named `string` and assign to it the value of `num`. Then we evaluate `string.charAt(0).toUpperCase() + string.slice(1,7);` and return that value from the function.

Alternatively, we could rename the argument:

```js
function capitalize(string) {
  return string.charAt(0).toUpperCase() + string.slice(1,7);
}
```